services/server/fixtures/mockApi.ts

```diff
@@ -1,4 +1,4 @@
-import Express from "express";
+import * as ExpressLib from "express";
```

Comment:
I belive there is no need for this. This pattern is not common in our codebase.

```suggestion
import {Request, Response, NextFunction} from "express";
```

Reply:
I have encountered an absurd TS error. This workaround allows me to use the generic functionality in the return value of the mockApi function:

I have no clue how to solve the problem otherwise.
```TS
export function mockApi<Body = object, Params = object, Query = object>(
  handler: (
    req: ExpressLib.Request,
    res: ExpressLib.Response,
    next: ExpressLib.NextFunction
  ) => Promise<void> | void
) {
  return async <Response>(
    request: MockRequest<Body, Params, Query>
  ): Promise<{ status: number | null; body: Response | null }> => {
...
```